### ISSUE TITLE
Counting comments

### ISSUE THREAD

**Issue by marconoris** [lukeleppan/better-word-count#82]
Hi, it seems plugin count comments although in the options it is indicated not to do it

**Comment by JVArcher** [comment]
I've just experienced the same issue. 

<img width="1030" alt="SCR-20230404-jmta" src="https://user-images.githubusercontent.com/129884660/229859202-ed5024db-1f50-42eb-b93f-c124b4ab5229.png">

**Comment by nicolascoffman** [comment]
#61 was closed but seems like it is the seem recurrent issue.

**Comment by nomad101** [comment]
Yes, I also see comments (wrapped by %% ... %%) are still being counted

**Comment by workingja** [comment]
I'm experiencing the same issue as of 2023-06-21. Hoping for a fix (plus a feature to exclude metadata in counting).

**Comment by mgmeyers** [comment]
This has been fixed in v0.10.0

**Comment by marconoris** [comment]
Thanks @mgmeyers... I guess would be important a feature to exclude metadata in counting too: there are very frequent cases in which you have to write texts with character limits and frontmatter should be excluded by default.

**Comment by hsandt** [comment]
I confirm fix for comments. We should close this issue then.

If you need to exclude more things you can add details to https://github.com/lukeleppan/better-word-count/issues/84

**Comment by MitchWagnerSD** [comment]
I am still seeing this bug in 0.10.0.

**Comment by scottzirkel** [comment]
Looks like this is still counting comments in the API. My counts via the API are different from the status bar. API seems to also count all frontmatter. A fresh note with frontmatter and comments should be a zero count.

``` markdown 
---
front:
matter:
stuff:
---
%%
Comments
%%
```

I am using the `getWordCountPagePath()` method in dataviewjs.

**Comment by skaviouz** [comment]
# V0.10.1
It's partial, maybe someone mixed up which "comments" are counted and omitted?

## What works correctly in V0.10.1
```
<!---
HTML Comments
-->
```
```
%% There are Comments between two Double Percents
   Allows multi-line and/or in-line
%%
```
```
$% [[Another Comment that works]]$
```

```
  %% Comment
%%%% Multi-line
%%%% Excessive
%%
```

## What does not work as expected in V0.10.1
```
%% Not counted
%% Counted, because multi-line depends on Obsidian? I swear I thought this one worked in the past?
```

````
```
# Codeblock Comments
```
````
```
[comment]: <> (Markdown Comment, universally compatible)
```
```
[//]: <> (Markdown Comment, alternative.)
```
```
[//]: # (Markdown Comment, compact)
```

## Conclusion:
Some ways work, some don't. I think it depends on Obsidian and not the plugin on what's recognized and what isn't counted or omitted. Unless you are using a particular commenting method specific for a reason, which I seriously doubt, just change how you write comments, there are three normal options, and the one that goes between `%%` and `%%` seems like the easiest to type and forget.
